components/meter/meter-radial.js

```diff
@@ -37,18 +42,22 @@ class MeterRadial extends LitElement {
 	render() {
 		const lengthOfLine = 115; // found by approximating half the perimeter of the ellipse with radii 38 and 35
 		const progressFill = this.value / this.max * lengthOfLine;
+		const secondaryTextElement = this.text ? html`<div class="d2l-body-small">${this.text}</div>` : html``;
```

Comment:
The text is exactly `d2l-body-small` it also requires `var(--d2l-color-ferrite)`.
You can modify `.d2l-meter-radial-text` and add `color: var(--d2l-color-ferrite)` since it will remind people they need to change both the fill and the color.